### DISCUSSION TITLE
How can I create application menu and Desktop shortcuts for games?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#462] [Q&A] [answered]
If I right click a game,I can't see that option.

**Comment by tkashkin** [comment, marked as answer]
There's currently no option for this, but it may be useful.

You can launch a game with GameHub using `com.github.tkashkin.gamehub --run <gameid>` command.
You can create `.desktop` files with this command manually and place them in `~/.local/share/applications`.

**Comment by ghost** [comment]
Did a [feature request](https://github.com/tkashkin/GameHub/issues/479) so other developers can see it too.
